### DISCUSSION TITLE
A grid as background for the waveform

### DISCUSSION THREAD

**Discussion by ManimMadhav** [katspaugh/wavesurfer.js#3607] [Q&A]
Hi all, I've been trying to implement a grid-like structure for the background of my waveform. It looks like this right now:<br>
![image](https://github.com/katspaugh/wavesurfer.js/assets/66988314/10ae63c2-80f0-4129-8c90-61ed279d1421)
I have added a grid that makes it look like this:<br>
![image](https://github.com/katspaugh/wavesurfer.js/assets/66988314/2f8a99d3-ff60-46be-a686-f9928c05ae80)
<br>
However, since this is hard-coded CSS that creates the grid, how can I make a grid background that renders when a user zooms in/out? is there any way to create a background like that?

This is the CSS code:
```
#audiowave {
    background-size: 40px 40px;
    background-image:
      linear-gradient(to right, grey 1px, transparent 1px),
      linear-gradient(to bottom, grey 1px, transparent 1px);
 }
```

**Comment by katspaugh** [comment]
If you want your grid to also scale, you should subscribe to wavesurfer’s zoom event and update the background-size based on the zoom level.

**Reply by ManimMadhav** [reply]
okay, i will try that. thanks for the response!
